docs/dg/dev/sdks/sdk/development-tools/phpstorm-support.md

```diff
@@ -0,0 +1,49 @@
+---
+title: PHPStorm Support
```

Comment:
What's the official name of the tools? Spryker Development Tools?

Reply:
We are planning to have multiple PhpSTorm support tools (they are still in the backlog), so this is why i named this page like this, so we can have the upcoming tools here as well. 

So this script, enables the existing Spryker SDK tools in PhpStorm. I don't know what is the best way to express this.